tests/attr/neuron/test_neuron_ablation.py

```diff
@@ -156,9 +156,9 @@ def test_simple_multi_input_conv(self):
             ablations_per_eval=(1, 3, 7, 14),
         )
 
-    def test_simple_multi_input_conv_intermediate(self):
+    def test_simple_multi_input_conv_intermediate(self) -> None:
         net = BasicModel_ConvNet_One_Conv(inplace=True)
-        inp = torch.arange(16).view(1, 1, 4, 4).type(torch.FloatTensor)
+        inp = torch.arange(16).view(1, 1, 4, 4)
```

Comment:
Removing the type will change behavior, can you instead update this line and above to 
`torch.arange(16, dtype=torch.float).view(1, 1, 4, 4)`